README.md

```diff
@@ -1,17 +1,27 @@
-Welcome to the SFOSC! This is the place to contribute content to the website,
-and collaborate on writing new social contracts, documenting business models,
-and more.
+[![content license CC BY-SA 4.0](https://badgen.net/badge/content%20license/CC%20BY-SA%204.0)](https://github.com/sfosc/sfosc/blob/master/LICENSE.md)
+[![code license MIT](https://badgen.net/badge/code%20license/MIT)](https://github.com/sfosc/sfosc/blob/master/LICENSE.md)
+[![discord](https://img.shields.io/discord/587972813302792217.svg?label=discord&logo=discord&logoColor=white)](https://discord.gg/nz5NC9q)
 
-[You might rather see the site in its final, rendered form](https://sfosc.org) - it's easier to read that way. 
+![SFOSC](static/logo.png)
+
+Welcome to the Sustainable Free and Open Source Community!
+This is the place to contribute content to the website, and collaborate
+on writing new social contracts, documenting business models, and more.
+
+The latest release is visible on https://sfosc.org - it's easier to read that way.<br/>
+The current `master` branch is visible on https://sfosc.org/preview
```

Comment:
One (unrelated) question - it looks like https://www.sfosc.org/ isn't working - who set up the DNS and can we fix this up?

Reply:
Good find. Poking with dig and checking github pages docs, indeed that would be just DNS settings to fix. I'm assuming just Adam currently has access to this.